### DISCUSSION TITLE
Overview in Gnome 4x does not work.

### DISCUSSION THREAD

**Discussion by quantenzitrone** [ppy/osu#17378] [Q&A]
When I press the Super-Key on my Gnome 40.5 Desktop (Ubuntu 21.10) from within Osu!, nothing happens.
However, switching windows via Hotkey works and when I go to Overview from another Application, Osu! appears there like every other Window.
The Problem is when no other Windows are open, I can only escape Osu! by opening another Application via Hotkey or quitting Osu!.

Other Fullscreen Games like Minecraft or Windows Games over Wine do not have this Problem. Though I must admit, Wine Games have other Problems with the Overview.

**Comment by bdach** [comment]
what window manager is this?

**Reply by quantenzitrone** [reply]
Gnome uses Mutter

**Reply by bdach** [reply]
and i presume this is with wayland by default now?

does this happen with the `SDL_VIDEODRIVER=x11` envvar set?

**Reply by quantenzitrone** [reply]
I don't know what `SDL_VIDEODRIVER=x11` envvar exactly means, but I'm on X11.

**Reply by bdach** [reply]
thanks for the info. this is going to be a problem that's specific to the distro/window manager and one that is going to most likely be in SDL rather than in the game itself, so i cannot promise this will be resolved anytime soon as i would have to reproduce your setup to investigate the bug which takes a considerable amount of time.
